homework/index.js

```diff
@@ -30,18 +31,102 @@
     return elem;
   }
 
+  function creatHeader(repositories, root) {
+    const header = createAndAppend('header', root, { class: 'header' });
+    createAndAppend('h2', header, {
+      text: 'HYF repositories',
+      class: 'nav-title',
+    });
+    const select = createAndAppend('select', header, { id: 'repo-selector' });
+    repositories.forEach((repo, index) => {
+      createAndAppend('option', select, { value: index, text: repo.name });
+    });
+    select.addEventListener('change', event => {
+      const selectedIndex = event.target.value;
+      const selectedRepository = repositories[selectedIndex];
+      while (mainContainerForRepo.firstChild) {
+        mainContainerForRepo.removeChild(mainContainerForRepo.firstChild);
+      }
+      createDescription(selectedRepository, root);
+      createContributors(selectedRepository, root);
+    });
+  }
+  const mainContainerForRepo = document.querySelector('.container');
+
+  function createDescription(selectedRepository) {
+    const descriptionContainer = createAndAppend('div', mainContainerForRepo, {
+      class: 'left-div',
+    });
+    const table = createAndAppend('table', descriptionContainer, { class: 'table' });
+    const tbody = createAndAppend('tbody', table);
+    const details = [
+      { title: 'Repository', value: selectedRepository.name },
+      { title: 'Description', value: selectedRepository.description },
+      { title: 'Forks', value: selectedRepository.forks },
+      { title: 'Updated', value: selectedRepository.updated_at },
+    ];
+    details.forEach(detail => {
+      const tr = createAndAppend('tr', tbody);
+      createAndAppend('td', tr, {
+        text: detail.title,
+        class: `label `,
+      });
+      createAndAppend('td', tr, { text: detail.value, class: 'repository-data' });
+    });
+  }
+
+  function createContributors(selectedRepository, root) {
+    const contributorsContainer = createAndAppend('div', mainContainerForRepo, {
+      class: 'right-div',
+    });
+    createAndAppend('h3', contributorsContainer, {
+      text: 'Contributors',
+      class: 'contributors-header',
+    });
+    const ul = createAndAppend('ul', contributorsContainer, { class: 'contributor-list' });
+    fetchJSON(selectedRepository.contributors_url, (err, ShowContributors) => {
+      if (err) {
+        createAndAppend('div', root, { text: err.message, class: 'alert-error' });
+      } else {
+        ShowContributors.forEach(contributor => {
+          const li = createAndAppend('li', ul, {
+            class: 'contributor-item',
+          });
+          const alink = createAndAppend('a', li);
```

Comment:
You are creating an `<a>` tag here, but it doesn't allow you to go to the contributor page, because you are not setting its `href` attribute.